packages/sitecore-jss-nextjs-editing-host/src/editing-middleware.ts

```diff
@@ -1,22 +1,29 @@
+// eslint-disable-next-line spaced-comment
 /// <reference types="../global" />
 import { parse } from 'url';
 import { Request, Response } from 'express';
 import Server from 'next/dist/next-server/server/next-server';
 import { EditingData } from '@sitecore-jss/sitecore-jss-nextjs';
 import { HtmlProcessor } from './html-processors';
-
-/**
- * Express middleware for handling requests from the Sitecore Experience Editor.
- * @constructor
- * @param {object} nextApp The Next.js app.
- * @param {string} editRoute The Next.js route to use for rendering.
- * @param {HtmlProcessor[]} [htmlProcessors] Html processors to run on rendered html.
- */
 export class EditingMiddleware {
-  constructor(readonly nextApp: Server, readonly editRoute: string, readonly htmlProcessors?: HtmlProcessor[]) {}
+  /**
+   * Express middleware for handling requests from the Sitecore Experience Editor.
+   * @constructor
+   * @param {Server} nextApp The Next.js app.
+   * @param {string} editRoute The Next.js route to use for rendering.
+   * @param {HtmlProcessor[]} [htmlProcessors] Html processors to run on rendered html.
+   */
+  constructor(
+    readonly nextApp: Server,
+    readonly editRoute: string,
+    readonly htmlProcessors?: HtmlProcessor[]
+  ) {}
 
   /**
    * Returns the Express request handler for Experience Editor POST requests.
+   * @param {Request} req
+   * @param {Response} res
+   * @returns {Promise<void>} request handler
```

Comment:
This JSDoc isn't accurate. There are no params, this is actually all part of the return, which would technically be `(req: Request, res: Response) => Promise<void>`. Think it would be fine to omit the return type here (since we've got that disabled for "valid-jsdoc").
```suggestion
   * @returns request handler
```

Reply:
@ambrauer Fixed it